### ISSUE TITLE
Scraping data issue

### ISSUE THREAD

**Issue by randoman00** [cpsievert/pitchRx#32]
When i try to download and send to mySQL database at some point during the season the table fields change and it creates a second copy of my data base. The code is below along with the error:

library(RMySQL);
drv <- dbDriver("MySQL
MLB <- dbConnect(drv, user="root" , password="password", port=3306, dbname="pitchfx", host="localhost")
scrape(start="2010-01-01", end="2010-10-30",connect = MLB)

http://gd2.mlb.com/components/game/mlb/year_2010/month_09/day_08/gid_2010_09_08_atlmlb_pitmlb_1/inning/inning_all.xml 
Successfully copied atbat table to database connection.
Successfully copied action table to database connection.
Successfully copied pitch table to database connection.
Successfully copied po table to database connection.
Successfully copied runner table to database connection.
Collecting garbage
Error in function (type, msg, asError = TRUE)  : 
  transfer closed with 1559 bytes remaining to read
In addition: Warning messages:
1: In export(connect, name = i, value = tables[[i]], template = fields[[i]]) :
  The value data.frame has variables that are not in the corresponding table. Writing data.frame to a new table instead.
2: In export(connect, name = i, value = tables[[i]], template = fields[[i]]) :
  The value data.frame has variables that are not in the corresponding table. Writing data.frame to a new table instead.

**Comment by dlependorf** [comment]
I'm getting a similar warning message as well, but I'm writing to a brand new SQLite file.

```r
library(RSQLite)
drv <- dbDriver("SQLite")
con <- dbConnect(drv,"./database_2013.sqlite3")

scrape(start="2013-01-01",end="2013-12-31",connect=con)
```
The atbat table fails to copy correctly, and it writes to a csv in the same directory instead. It returns this message:

> Failed to copy atbat_export table to database connection. Writing atbat_export-2015-09-07-17-56-01.csv instead.
Successfully copied action_export table to database connection.
Successfully copied pitch_export table to database connection.
Successfully copied po_export table to database connection.
Successfully copied runner table to database connection.
Collecting garbage

If I look at the warning messages, I get a bunch of warnings like this:

> Warning messages:
1: In export(connect, name = i, value = tables[[i]], template = fields[[i]]) :
  The value data.frame has variables that are not in the corresponding table. Writing data.frame to a new table instead.
2: In export(connect, name = i, value = tables[[i]], template = fields[[i]]) :
  The value data.frame has variables that are not in the corresponding table. Writing data.frame to a new table instead.
3: In export(connect, name = i, value = tables[[i]], template = fields[[i]]) :
  The value data.frame has variables that are not in the corresponding table. Writing data.frame to a new table instead.

**Comment by cpsievert** [comment]
@randoman00 @dlependorf what version of pitchRx are you using (you can check with `packageVersion("pitchRx")`)? If you have 1.7 or lower, could you please install the current GitHub version (you can do that with `devtools::install_github('cpsievert/pitchRx')`)?

**Comment by randoman00** [comment]
I will try that.

**Comment by dlependorf** [comment]
Much thanks, Carson, I was on 1.7 from CRAN. Ran a full scrape using the dev version of 1.8, and didn't run into a single error or warning.

**Comment by BillPetti** [comment]
I was actually having the same issues, so I tried installing the most recent version and am now running into another issue:

`> devtools::install_github('cpsievert/pitchRx')`
`> require(pitchRx)`
`Loading required package: pitchRx
Failed with error:  ‘lazy-load database '/Library/Frameworks/R.framework/Versions/3.2/Resources/library/pitchRx/data/Rdata.rdb' is corrupt’
In addition: Warning messages:
1: In FUN(X[[i]], ...) : restarting interrupted promise evaluation
2: In FUN(X[[i]], ...) : internal error -3 in R_decompress1`

**Comment by cpsievert** [comment]
@BillPetti that error usually means you have to restart R, then re-install

**Comment by cpsievert** [comment]
For the record, I think this issue is a duplicate of #26 which was fixed in 59885bdc68a0a3a9940783c65c0cfe66ea20399c
